src/gain-calculation.md

```diff
@@ -0,0 +1,302 @@
+# Calculate unrealized gain
+
+<div class=pagetoc>
+<!-- toc -->
+</div>
+
+This is a guide on calculating the unrealized gain using `--gain`
+while tracking investments in hledger. This guide assumes you've read
+[the investments guide](investments.md) and that you're using the
+"simple" version of recording investment transactions laid out in that
+document, using @ or @@. We'll also be using a FIFO system for
+sales. At the end we'll discuss how to adapt your strategy for
+different systems.
+
+## Buying
+
+Let's say you start your year with $100:
+
+```
+2021-01-04 opening balances
+    assets:cash                      $100.00
+    equity:opening/closing balances
+```
+
+In february you decide to start investing in a stock you've been
+monitoring:
+
+```
+P 2021-01-04 ABC $2
+P 2021-01-11 ABC $3
+P 2021-01-18 ABC $4
+
+2021-01-19 buying stock
+    assets:stocks:ABC:20210119   5 ABC @ $4.40
+    assets:cash               $-22.00
+```
+
+Note that the stock was not bought at the exact market price we
+recorded.
+
+Executing `hledger bal --gain` at this point tells us that we've lost
+$2:
+
+```
```

Comment:
Consider including the command in this and the other examples ? I always try to begin these with the exact command (and `$` shell prompt), to make them clearer and more reproducible.

Reply:
Done.